include/vkma_bindings.hpp

```diff
@@ -1,694 +1,693 @@
-                 reinterpret_cast<VmaVirtualAllocation>(allocation));
-}
-inline void vkmaClearVirtualBlock(VkmaVirtualBlock virtualBlock) {
-  vmaClearVirtualBlock(reinterpret_cast<VmaVirtualBlock>(virtualBlock));
-}
-
-inline void vkmaSetVirtualAllocationUserData(VkmaVirtualBlock virtualBlock,
-                                             VkmaVirtualAllocation allocation, void *pUserData) {
-  vmaSetVirtualAllocationUserData(reinterpret_cast<VmaVirtualBlock>(virtualBlock),
-                                  reinterpret_cast<VmaVirtualAllocation>(allocation), pUserData);
-}
-inline void vkmaGetVirtualBlockStatistics(VkmaVirtualBlock virtualBlock, VkmaStatistics *pStats) {
-  vmaGetVirtualBlockStatistics(reinterpret_cast<VmaVirtualBlock>(virtualBlock),
-                               reinterpret_cast<VmaStatistics *>(pStats));
-}
-inline void vkmaCalculateVirtualBlockStatistics(VkmaVirtualBlock virtualBlock,
-                                                VkmaDetailedStatistics *pStats) {
-  vmaCalculateVirtualBlockStatistics(reinterpret_cast<VmaVirtualBlock>(virtualBlock),
-                                     reinterpret_cast<VmaDetailedStatistics *>(pStats));
-}
-inline void vkmaBuildVirtualBlockStatsString(VkmaVirtualBlock virtualBlock, VkBool32 detailedMap,
-                                             char **ppStatsString) {
-  vmaBuildVirtualBlockStatsString(reinterpret_cast<VmaVirtualBlock>(virtualBlock), ppStatsString,
-                                  detailedMap);
-}
-inline void vkmaFreeVirtualBlockStatsString(VkmaVirtualBlock virtualBlock,
-                                            const char *pStatsString) {
-  vmaFreeVirtualBlockStatsString(reinterpret_cast<VmaVirtualBlock>(virtualBlock),
-                                 const_cast<char *>(pStatsString));
-}
-
-inline void vkmaBuildStatsString(VkmaAllocator allocator, VkBool32 detailedMap,
-                                 char **ppStatsString) {
-  vmaBuildStatsString(reinterpret_cast<VmaAllocator>(allocator), ppStatsString, detailedMap);
-}
-inline void vkmaFreeStatsString(VkmaAllocator allocator, const char *pStatsString) {
-  vmaFreeStatsString(reinterpret_cast<VmaAllocator>(allocator), const_cast<char *>(pStatsString));
-}
-
-inline void vkmaGetBuffer(VkmaBuffer buffer, VkBuffer *pBuffer) {
-  VKMA_ASSERT(pBuffer);
-  *pBuffer = reinterpret_cast<VkmaBuffer_T *>(buffer)->buffer;
-}
-inline void vkmaGetBufferAllocation(VkmaBuffer buffer, VkmaAllocation *pAllocation) {
-  VKMA_ASSERT(pAllocation);
-  *pAllocation = reinterpret_cast<VkmaBuffer_T *>(buffer)->allocation;
-}
-inline void vkmaGetImage(VkmaImage image, VkImage *pImage) {
-  VKMA_ASSERT(pImage);
-  *pImage = reinterpret_cast<VkmaImage_T *>(image)->image;
-}
-inline void vkmaGetImageAllocation(VkmaImage image, VkmaAllocation *pAllocation) {
-  VKMA_ASSERT(pAllocation);
-  *pAllocation = reinterpret_cast<VkmaImage_T *>(image)->allocation;
-}
-
-#endif // VKMA_BINDINGS_HPP
+#ifndef VKMA_BINDINGS_HPP
```

Comment:
Could you also squash both of these commits together please?

Because of EoL changes it's rather hard to review them.

Reply:
Ah, sorry about that; should be squashed now.